src-overlay-ui/styles/_tokens.scss

```
@@ -0,0 +1,36 @@
+$gray-700: #374151;
+$gray-800: #1f2937;
+$cyan-400: #22d3ee;
+$cyan-500: #06b6d4;
+$cyan-600: #0891b2;
+$cyan-700: #0e7490;
+$red-400: #f87171;
+$red-500: #ef4444;
+$red-600: #dc2626;
+$red-700: #b91c1c;
+
+$ease-cubic-out: cubic-bezier(0.33, 1, 0.68, 1);
```

Stylelint (17.14.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)